db/migrations/20211019191617-create-question.js

```diff
@@ -21,10 +21,18 @@ module.exports = {
       },
       categoryId: {
         type: Sequelize.INTEGER,
+        references: {
+          model: "Category",
+          key: "id",
+        },
       },
       userId: {
         allowNull: false,
         type: Sequelize.INTEGER,
+        references: {
+          model: "User",
+          key: "id",
```

Comment:
I don't think you need to add the key: "id" because it should default to the PK on the user table